Program.cs

```diff
@@ -8,6 +8,7 @@ namespace Algorithms
 {
 	class Program
 	{
+        //tambah comment ga jelas di program.cs
```

Comment:
Just a comment don't do :sweat_smile: 

Please kindly contribute an algorithm or optimization. You are welcome, :100: